extras/iris_pipeline_project/pipeline/ovmsmodel.py

```diff
@@ -0,0 +1,141 @@
+import os
+import sys
+import json
+import numpy as np
+import pandas as pd
+from sklearn.linear_model import LogisticRegression
+from skl2onnx import convert_sklearn
+from skl2onnx.common.data_types import FloatTensorType
+import onnxruntime as ort
+from pyovms import Tensor
+from io import StringIO
+
+from sklearn.metrics import accuracy_score, precision_score, recall_score, f1_score, classification_report
+
+MODEL_PATH = "/workspace/model/iris_logreg/1/model.onnx"
+LABEL_COLUMN = "Species"
+DROP_COLUMNS = ["Id"]
+
+class OvmsPythonModel:
+    def initialize(self, kwargs):
+        print("[initialize] Python node initialized", file=sys.stderr, flush=True)
+
+    def execute(self, inputs):
+        print("==== [execute] Python node called ====", file=sys.stderr, flush=True)
+        try:
+            input_tensor = inputs[0]
+            input_data = input_tensor.data
+            if isinstance(input_data, memoryview):
+                inp_bytes = bytes(input_data)
+            elif isinstance(input_data, (bytes, bytearray)):
+                inp_bytes = input_data
+            elif isinstance(input_data, np.ndarray):
+                if input_data.dtype == object:
+                    inp_bytes = input_data[0]
+                else:
+                    inp_bytes = bytes(input_data)
```

Comment:
`input_tensor.data` will always be a memoryview:
https://github.com/openvinotoolkit/model_server/blob/main/src/python/binding/ovms_py_tensor.cpp#L57
 
I think we can safely get rid of that condition. That object is provided by OVMS, so it must be okay or it would throw error earlier in the model server logic.